Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -12,7 +12,7 @@ public class CBPurchase: NSObject {
     public static let shared = CBPurchase()
     private var productIDs: [String] = []
     public var receiveProductsHandler: ((_ result: Result<[CBProduct], CBPurchaseError>) -> Void)?
-    public var buyProductHandler: ((Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)?
+    public var buyProductHandler: ((Result<(status:Bool, subscriptionId:String?, planId:String?, customerId:String?), Error>) -> Void)?
```

Comment:
This will be a breaking change for the SDK. If its not related to this PR, can we revert and take it up separately?

Reply:
> This will be a breaking change for the SDK. If its not related to this PR, can we revert and take it up separately?

ok Its reverted